src/commonmark.rs

```diff
@@ -440,6 +440,68 @@ impl<'a, W: Write + 'a> Context<'a, W> {
         );
         write!(self.output.writer, "{}", osc(&command))
     }
+
+    /// Write an inline image for Terminology.
+    fn write_terminology_inline_image(&mut self, path: &Path) -> Result<()> {
+        fn get_lines(columns: u16, file: &str) -> u16 {
+            use std::process::Command;
+            use termion::terminal_size;
+
+            let lines = terminal_size().unwrap_or((0, 24)).1;
+
+            Command::new("identify")
```

Comment:
Can't we use a crate here?  Perhaps `image`?   I'd prefer if we could work without a runtime dependency on imagemagick here.

If it's not possible, tho, please document that `identify` must be in `$PATH` for Terminology image support.

Reply:
We can use a crate here. But do you want to inflate the binary size with parsing code for multiple image formats in an application that is essentially used in terminals and it'd be a natural solution to depend on external commands?

Just choose an option and I'll go with it. I didn't add a crate because I thought you'd dislike this option.